plugins/node/opentelemetry-instrumentation-nestjs-core/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@opentelemetry/instrumentation-nestjs-core",
+  "version": "0.23.0",
+  "description": "OpenTelemetry NestJS automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "compile": "npm run version:update && tsc -p .",
+    "compile:watch": "tsc -w",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "version:update": "node ../../../scripts/version-update.js"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nestjs",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
```suggestion
    "node": ">=8.5.0"
```

Reply:
There are quite a few packages that have `>=8.0.0` as the engine requirement. I'll make a note to have a PR to bump those to the actual supported version as well!